packages/js/src/shared-admin/components/premium-upsell-card.js

```diff
@@ -72,9 +76,13 @@ export const PremiumUpsellCard = ( { link, linkProps, promotions } ) => {
 				className="yst-flex yst-justify-center yst-gap-2 yst-mt-4 focus:yst-ring-offset-primary-500"
 				{ ...linkProps }
 			>
-				<span>{ isBlackFriday ? __( "Claim your 30% off now!", "wordpress-seo" ) : getPremium }</span>
+				<span>{ isBlackFriday ? __( "Claim your 30% off now!", "wordpress-seo" ) : explorePremium }</span>
 				<ArrowNarrowRightIcon className="yst-w-4 yst-h-4 yst-icon-rtl" />
 			</Button>
+			<p className="yst-text-center yst-text-[11px] yst-leading[19px] yst-mt-2">
```

Comment:
I'm not liking these specific overrides to the normal EM mechanisms in Tailwind. Are you sure this is needed?
E.g. did you try `yst-text-xs` etc. to see if you arrive at the wanted pixel size?

Also, the leading won't do anything without that dash there:
```suggestion
			<p className="yst-text-center yst-text-[11px] yst-leading-[19px] yst-mt-2">
```